http-verbs-common/src/main/scala/uk/gov/hmrc/http/HttpGet.scala

```diff
@@ -27,30 +25,15 @@ import scala.concurrent.{ExecutionContext, Future}
 trait HttpGet extends CoreGet with GetHttpTransport with HttpVerb with ConnectionTracing with HttpHooks with Retries {
 
   override def GET[A](
-    url: String,
-    queryParams: Seq[(String, String)],
-    headers: Seq[(String, String)])(
-      implicit rds: HttpReads[A],
-      hc: HeaderCarrier,
-      ec: ExecutionContext): Future[A] = {
-    if (queryParams.nonEmpty && url.contains("?")) {
```

Comment:
This logic is not needed as the query params in UrlBuilder will overwrite the duplicate query param in the url